doc/message-registry.md

```diff
@@ -6,7 +6,7 @@ NOTE: This file has been generated automatically by “update-message-registry.p
 -->
 
 This page lists the warnings and errors generated by xkbcommon.
-There are currently 54 entries.
```

Comment:
Is this correct? Looks like 428 was replaced with a different error so shouldn't the number be the same?

Reply:
I just re-run the script:  it is correct. It seems that it was not updated properly in 00e3058e7b027c3d698b24415fd2ac105cd72246. Probably because of an incomplete rebase.

Reply:
might be worth splitting that particular update out if you have the time and motivation to do it, otherwise just note it in the commit message please, thanks.

Reply:
Good point. Done